core/lib.rs

```diff
@@ -2403,3 +2403,15 @@ impl Iterator for QueryRunner<'_> {
         }
     }
 }
+
+impl Value {
+    pub fn to_display(&self) -> String {
+        match self {
+            Value::Null => "NULL".to_string(),
+            Value::Integer(i) => i.to_string(),
+            Value::Float(f) => crate::numeric::f64_to_string(*f),
+            Value::Text(t) => t.to_string(),
+            Value::Blob(b) => format!("BLOB({} bytes)", b.len()),
+        }
+    }
+}
```

Comment:
just call `to_string` of `Display` trait, i think `f64_to_string` should be in this function https://github.com/tursodatabase/turso/blob/1de647758f40266451243c85ab56f8d21df29ca2/core/types.rs#L428